src/sw.ts

```diff
@@ -177,13 +186,93 @@ function isSwAssetRequest (event: FetchEvent): boolean {
   return isActualSwAsset
 }
 
+/**
+ * Set the expires header on a response object to a timestamp based on the passed ttl interval
+ */
+function setExpiresHeader (response: Response, ttlSeconds: number = 3600): void {
+  const expirationTime = new Date(Date.now() + ttlSeconds * 1000)
+
+  response.headers.set('Expires', expirationTime.toUTCString())
```

Comment:
It may be sensible to make this cache header explicitly SW specific (originally suggested in https://phyks.me/2019/01/manage-expiration-of-cached-assets-with-service-worker-caching.html)
```suggestion
  response.headers.set('sw-cache-expires', expirationTime.toUTCString())
```

Reply:
that makes sense

Reply:
If we can avoid inventing things specific to service worker, and reuse HTTP semantics, that is better for verified-fetch being useful in more contexts.

My initial thought here was that if we reuse semantically meaningful `Expires` and `Cache-Control`, in the future these can be set inside `verified-fetch`, and not only we don't need to invent special header here, but also improve the upstream library and the code here will be smaller.

Reply:
@lidel Because the SW Cache API always takes precedence over browser HTTP cache and and `verified-fetch` [will be returning the `Cache-Control` header](https://github.com/ipfs/helia-verified-fetch/pull/19), the `Expires` header would be: no-op. 

Even if we were to use the `Expires` header, by respecting it when there's a `Cache-Control` header, we'd be breaking from HTTP semantics. This is why I think it makes sense to make it clear that this header is purely for SW purposes.